app/policies/rent_policy.rb

```diff
@@ -0,0 +1,19 @@
+class RentPolicy < ApplicationPolicy
+  class Scope
```

Comment:
You need Scope class?

Reply:
Yes, for policy scopes: https://github.com/varvet/pundit#scopes

Reply:
Really nice! I've forgotten it :angel: , thanks!